src/ComposerRequireChecker/Cli/CheckCommand.php

```diff
@@ -73,14 +68,17 @@ protected function execute(InputInterface $input, OutputInterface $output) : int
             )
         ));
 
-        $options = $this->getCheckOptions($input);
-
         $definedExtensionSymbols = (new LocateDefinedSymbolsFromExtensions())->__invoke(
             (new DefinedExtensionsResolver())->__invoke($composerJson, $options->getPhpCoreExtensions())
         );
 
         $usedSymbols = (new LocateUsedSymbolsFromASTRoots())->__invoke($sourcesASTs($getPackageSourceFiles($composerJson)));
 
+        if (!count($usedSymbols)) {
+            $output->writeln("There were no symbols found, please check your configuration.");
```

Comment:
Throw an exception - easier to see traces when using `-vvv`. Also easier to test